operators/volume_loader/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ target_link_libraries(volume_loader
   PRIVATE
     holoscan::core
     NIFTI::nifti2
+    pybind11::embed
```

Comment:
I think this is no longer needed, I assume this was used for pre 2.1 not supporting Python type conversions for C++ operators.